scripts/rook.sh

```diff
@@ -8,6 +8,21 @@ ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
 KUBECTL_RETRY=5
 KUBECTL_RETRY_DELAY=10
 
+trap log_errors EXIT
+
+# log_errors is called on exit (see 'trap' above) and tries to provide
+# sufficient information to debug deployment problems
+function log_errors() {
+	# enable verbose execution, disable exit on failure
+	set -x +e
+	kubectl get nodes
```

Comment:
Don't you want the operator logs too?

Reply:
Yes, maybe. Currently most issues are probably due to Docker Pull limits, but the operator logs would be helpful in other cases.